src/components/vm/hostdevs/hostDevCard.jsx

```diff
@@ -0,0 +1,175 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2021 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import cockpit from 'cockpit';
+import { vmId, findHostNodeDevice } from "../../../helpers.js";
+import { ListingTable } from "cockpit-components-table.jsx";
+
+const _ = cockpit.gettext;
+
+/* Adds an optional description-value pair to an array which represents multiple values of a table cell
+ *
+ * @param chunks an array into which element is added to
+ * @param value a value of the descr-value pair
+ * @param descr a description of the descr-value pair
+ * @param id
+ */
+function addOptionalToCell(chunks, value, id, descr) {
```

Comment:
This function is copied from somewhere (disks?) right?
Let's move it to helpers or some common file and reuse it instead.